CHANGELOG.md

```diff
@@ -1,5 +1,35 @@
 # Changelog
 
+## 14.11.0
+
+### Minor Changes
+
+- Added: `ignoreAfterCombinators: []` to `selector-max-universal` ([#6275](https://github.com/stylelint/stylelint/pull/6275)) ([#6274](https://github.com/stylelint/stylelint/pull/6274)) ([@JounQin](https://github.com/JounQin)).
+
+### Patch Changes
+
+- Fixed: `selector-pseudo-class-allowed-list` end positions and message ([#6262](https://github.com/stylelint/stylelint/pull/6262)) ([#6274](https://github.com/stylelint/stylelint/pull/6274)) ([@JounQin](https://github.com/JounQin)).
+
+* Fixed: `selector-no-vendor-prefix` end positions ([#6261](https://github.com/stylelint/stylelint/pull/6261)) ([#6274](https://github.com/stylelint/stylelint/pull/6274)) ([@JounQin](https://github.com/JounQin)).
+
+- Fixed: `shorthand-property-no-redundant-values` message ([#6272](https://github.com/stylelint/stylelint/pull/6272)) ([#6274](https://github.com/stylelint/stylelint/pull/6274)) ([@JounQin](https://github.com/JounQin)).
+
+* Fixed: `selector-pseudo-element-disallowed-list` end positions and message ([#6270](https://github.com/stylelint/stylelint/pull/6270)) ([#6274](https://github.com/stylelint/stylelint/pull/6274)) ([@JounQin](https://github.com/JounQin)).
+
+- Fixed: `selector-pseudo-element-allowed-list` end positions and message ([#6270](https://github.com/stylelint/stylelint/pull/6270)) ([#6274](https://github.com/stylelint/stylelint/pull/6274)) ([@JounQin](https://github.com/JounQin)).
+
+* Fixed: `time-min-milliseconds` end positions ([#6273](https://github.com/stylelint/stylelint/pull/6273)) ([#6274](https://github.com/stylelint/stylelint/pull/6274)) ([@JounQin](https://github.com/JounQin)).
+
+- Fixed: `keyframes-name-pattern` false positives for interpolation ([#6265](https://github.com/stylelint/stylelint/pull/6265)) ([#6274](https://github.com/stylelint/stylelint/pull/6274)) ([@JounQin](https://github.com/JounQin)).
+
+* Fixed: `selector-nested-pattern` end positions ([#6259](https://github.com/stylelint/stylelint/pull/6259)) ([#6274](https://github.com/stylelint/stylelint/pull/6274)) ([@JounQin](https://github.com/JounQin)).
```

Comment:
If the second one (i.e. `(#6274)`) expected to be the related issue then the prefix should be `/issues/`.
Also it should be optional.

Reply:
The second one should be omitted before releasing, see also https://github.com/stylelint/stylelint/pull/6282#issuecomment-1219429530 and https://github.com/stylelint/stylelint/pull/6274#discussion_r948879544